src/gwkokab/models/utils/_lazyjointdistribution.py

```
@@ -0,0 +1,269 @@
+# Copyright 2023 The GWKokab Authors
+# SPDX-License-Identifier: Apache-2.0
+
+
+from collections.abc import Sequence
+from typing import Dict, List, Literal, Optional, Tuple, Union
```

Ruff (0.14.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)